website/blog/2022-12-01-release-0.10-blog.md

```diff
@@ -0,0 +1,113 @@
+
+**1. Custom Kubeconfig file path ([#780](https://github.com/containers/podman-desktop/pull/780))** 
+
+Starting v0.10.0, users can provide their kubeconfig path location instead of letting Podman Desktop use the default path i.e. `$HOME/.kube/config`. This can be done by going to `Settings > Preferences > Kubernetes: Kubeconfig`
+
+![custom kubeconfig file path](img/podman-desktop-release-0.10/custom-kubeconfig.png)
+
+**2. Play Kubernetes YAML from the Containers list ([#739](https://github.com/containers/podman-desktop/pull/739))**
+
+Development workflows can require developers to leverage existing Kubernetes YAML files. Podman Desktop now supports playing Pods or Containers from an existing Kubernetes YAML File. 
+
+This can be done in the ‘Containers’ screen of the application by clicking the `Play Kubernetes YAML` button and playing the existing YAML file.
+
+**3. Kube context automatically reloaded when updated ([#813](https://github.com/containers/podman-desktop/pull/813))**
+
+Previously, Podman Desktop captured the kube context only at startup. Any change to the kube context would not be detected after that. This required the user to restart Podman Desktop. With this change, one can now reload kube context without restarting Podman Desktop.
+
+Launch Podman Desktop and then try to change the current kube context either by using the tray icon or with `kubectl` tool and then try to use ‘Deploy to Kubernetes’ feature, the selected context should be displayed.
+
+## UI Improvements
+
+**1. Kebab menu ([#877](https://github.com/containers/podman-desktop/pull/877))**
+
+With the latest release, you will see significant changes with respect to the options and how they are displayed for images and containers. Previously, there was an array of buttons over each container or image component which has now been replaced with a ‘kebab’ menu. This is more user-friendly and keeps all options organized within a single component. By clicking the three dot button, you can open the menu and choose the option depending on your workflow.
+
+![kebab menu](img/podman-desktop-release-0.10/kebab-menu.png)
+
+**2. Containers ([#878](https://github.com/containers/podman-desktop/pull/878))**
+
+The list of containers now also displays the age of the container as well as the image used to start the container in separate columns alongside the container name.
+
+**3. Responsive Design ([#771](https://github.com/containers/podman-desktop/pull/771), [#774](https://github.com/containers/podman-desktop/pull/774))**
+
+On account of the effort to improve the User Experience, we have made the UI more responsive to window size changes. This includes animations as well as consistent design across all sections of the application.
+
+**4. Registries ([#747](https://github.com/containers/podman-desktop/pull/747))**
+
+We have also updated the UI of the page where we add/update registries in ‘Settings’.
```

Comment:
here, we have a feature which is pretty interesting. Out of the box we enable users to work with multiple registries. We should try to highlight it and giving it a bit more attention

Reply:
https://github.com/containers/podman-desktop/pull/943/commits/ed51bbcba5b04ecdae8a6406aabbd2040153097a